Tasks/AzureFunctionAppContainerV1/azurefunctiononcontainerprovider.ts

```diff
@@ -29,9 +30,16 @@ export class AzureFunctionOnContainerDeploymentProvider{
         this.azureEndpoint = await new AzureRMEndpoint(this.taskParams.connectedServiceName).getEndpoint();
         console.log(tl.loc('GotconnectiondetailsforazureRMWebApp0', this.taskParams.WebAppName));
         
-        if(!this.taskParams.ResourceGroupName){
-            var appDetails = await AzureResourceFilterUtility.getAppDetails(this.azureEndpoint, this.taskParams.WebAppName);
+        if(!this.taskParams.ResourceGroupName) {
```

Comment:
Should we move this code to taskparameters? For multi-container, we might do some checks on apptype before deployment provider is invoked.

Reply:
removed that logic, as function app container has only linux